src/index.js

```diff
@@ -15,17 +15,17 @@ const history = createBrowserHistory();
 const App = () => {
   const store = initStore(history);
   return (
-    <OverlayContextProvider>
-      <ProviderContextProvider>
-        <Provider store={store}>
-          <TokenInformationContextProvider>
-            <ConnectedRouter history={history}>
+    <Provider store={store}>
+      <ConnectedRouter history={history}>
+        <OverlayContextProvider>
+          <ProviderContextProvider>
+            <TokenInformationContextProvider>
```

Comment:
Can explain the reordering?

Reply:
wanted to access useLocation within context, will be reverting this as per discussed